jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
     testEnvironment: 'node',
     transform: {
-        '^.+\\.tsx?$': 'esbuild-jest',
+        '^.+\\.tsx?$': '@swc/jest',
```

Comment:
turns out you can't have mocks in tests because esbuild-jest doesn't allow `ock(` in any file

https://github.com/aelbore/esbuild-jest/issues/57

so, let's not use it